doc/source/io.rst

```diff
@@ -3965,72 +3982,173 @@ function.
+
+.. code-block:: python
+
+    df.to_gbq('my_dataset.my_table', projectid, chunksize=10000)
+
+You can also see the progress of your post via the ``verbose`` flag which defaults to ``True``.
+For example:
+
+.. code-block:: python
+
+    df.to_gbq('my_dataset.my_table', projectid, chunksize=10000, verbose=True)
+
+.. note::
+
+    If an error occurs while streaming data to BigQuery, see
+    `Troubleshooting BigQuery Errors <https://cloud.google.com/bigquery/troubleshooting-errors>`__.
+
+.. note::
+
+    The BigQuery SQL query language has some oddities, see the
+    `BigQuery Query Reference Documentation <https://developers.google.com/bigquery/query-reference>`__.
+
+.. note::
+
+    While BigQuery uses SQL-like syntax, it has some important differences from traditional
+    databases both in functionality, API limitations (size and quantity of queries or uploads),
+    and how Google charges for use of the service. You should refer to `Google BigQuery documentation <https://developers.google.com/bigquery/>`__
+    often as the service seems to be changing and evolving. BiqQuery is best for analyzing large
+    sets of data quickly, but it is not a direct replacement for a transactional database.
+
+
+Creating BigQuery Tables
+''''''''''''''''''''''''
+
+As of 0.17.0, the gbq module has a function :func:`~pandas.io.gbq.create_table` which allows users
+to create a table in BigQuery. The only requirement is that the dataset must already exist.
+The schema may be generated from a pandas DataFrame using the :func:`~pandas.io.gbq.generate_bq_schema` function below.
+
+For example:
+
+.. code-block:: python
+
+    gbq.create_table('my_dataset.my_table', schema, projectid)
+
+As of 0.15.2, the gbq module has a function :func:`~pandas.io.gbq.generate_bq_schema` which will
+produce the dictionary representation schema of the specified pandas DataFrame.
+
+.. code-block:: python
+
+    df = pd.DataFrame({'my_string': list('abc'),
+                  'my_int64': list(range(1, 4)),
+                  'my_float64': np.arange(4.0, 7.0),
+                  'my_bool1': [True, False, True],
+                  'my_bool2': [False, True, False],
+                  'my_dates': pd.date_range('now', periods=3).values})
+
+    In [5]: gbq.generate_bq_schema(df, default_type='STRING')
+
+    Out[5]: {'fields': [{'name': 'my_bool1', 'type': 'BOOLEAN'},
+          {'name': 'my_bool2', 'type': 'BOOLEAN'},
+          {'name': 'my_dates', 'type': 'TIMESTAMP'},
+          {'name': 'my_float64', 'type': 'FLOAT'},
+          {'name': 'my_int64', 'type': 'INTEGER'},
+          {'name': 'my_string', 'type': 'STRING'}]}
+
+Deleting BigQuery Tables
+''''''''''''''''''''''''
+
+As of 0.17.0, the gbq module has a function :func:`~pandas.io.gbq.delete_table` which allows users to delete a table
+in Google BigQuery.
+
+For example:
+
+.. code-block:: python
+
+    gbq.delete_table('my_dataset.my_table', projectid)
```

Comment:
There is a 'list' function in the BigQuery API which returns a list of tables under a specified dataset. The pandas.io gbq module doesn't currently support 'list'. This could be a useful feature for a future pandas release.
(https://cloud.google.com/bigquery/docs/reference/v2/tables/list)